spring-security/pom.xml

```diff
@@ -33,36 +33,19 @@
 
     <dependency>
       <groupId>org.mockito</groupId>
-      <artifactId>mockito-core</artifactId>
-      <scope>test</scope>
-    </dependency>
-
-    <dependency>
-      <groupId>org.powermock</groupId>
-      <artifactId>powermock-api-mockito2</artifactId>
-      <version>2.0.9</version>
+      <artifactId>mockito-inline</artifactId>
+      <version>4.5.1</version>
```

Comment:
@hennr I guess this should be removed and the dependency be in the parent module? In my mind I am still in the Gradle world.